kafkajs.d.ts

```diff
@@ -0,0 +1,261 @@
+/**
+ * Constructor types
+ */
+
+interface SSLConfiguration {
+    rejectUnauthorized: boolean;
+    ca: string[];
+    key: string;
+    cert: string;
+}
+export type SSLOptions = SSLConfiguration | boolean;
+export interface SASLOptions {
+    mechanism: 'plain' | 'scram-sha-256' | 'scram-sha-512';
+    username: string;
+    password: string;
+}
+export interface RetryOptions {
+    maxRetryTime?: number;
+    initialRetryTime?: number;
+    factor?: number;
+    multiplier?: number;
+    retries?: number;
+}
+export enum ELogLevel {
```

Comment:
Is it a convention in TS to prefix enum types with `E`? If it's not, I would prefer to omit that and just go with `LogLevel`, `CompressionTypes` etc.

Reply:
It's a convention I follow, although not sure if that's pervasive in the wider TS community. In this instance I prefixed it to avoid conflicts with the exported const's